roadmap/roadmap.md

```diff
@@ -0,0 +1,90 @@
+# The Astropy Roadmap
+
+- [Status Legend](#Status-Legend)
+- [Functionality](#Functionality)
+- [Hardware and Performance](#Hardware-and-Performance)
+- [Learn and User Support](#Learn-and-User-Support)
+- [Community Building and Sustainability](#Community-Building-and-Sustainability)
+- [Infrastructure, Documentation](#Infrastructure-Documentation)
+- [Government, Management, and Personnel](#government-management-and-personnel)
+
+
+## Status Legend
+
+### Green :green_circle:
+- Group or person leading effort is identified and sufficiently supported.
+- Effort is already underway and/or expected to be implemented in the “near term.”
+
+### Orange :large_orange_diamond:    
+- Item is already defined (e.g., issue, APE exists), but not sufficiently underway.
+- Needs more people and/or resources to make it happen. 
+  - *For example, funding applied for, but not awarded.*
+  
+### Red :red_square:  
+- Consensus of the community is that the item is a priority.
+- Item won’t happen without more people/resources.

```

Comment:
How is this different from “needs more people and/or resources to make it happen” (above)?

Reply:
The examples sort of make the distinction more clear, but I suggest rewording so the examples aren't necessary. Maybe something like:

(Orange): Attempts to secure the necessary resources are underway.
(Red): Attempts to secure the necessary resources have not yet been started.